src/odemis/acq/test/stream_test.py

```diff
@@ -4420,9 +4401,9 @@ def test_acquisition(self):
         self.assertIn(self.ebeam.dwellTime.value, (1, 1e-6))
         data, exp = f.result()
         self.assertIsNone(exp)
-        # Dwell time on detector and emitter should be back to normal
-        self.assertEqual(self.time_correlator.dwellTime.value, 2)
-        self.assertEqual(self.ebeam.dwellTime.value, 0.042)
+        # # Dwell time on detector and emitter should be back to normal
+        # self.assertEqual(self.time_correlator.dwellTime.value, 2)
+        # self.assertEqual(self.ebeam.dwellTime.value, 0.042)
```

Comment:
why is this commented?

Reply:
It should be completely dropped. We typically don't need the settings to be set back to the previous value. I've removed the code that does it in the stream, and so the test has to drop these checks.